Models/post-and-patch-models.js

```diff
@@ -0,0 +1,30 @@
+const { copy } = require('../app.js')
+const db = require('../db/connection.js')
+
+exports.insertNewComment = (newComment, articleId, author) => {
+    
+      if(isNaN(articleId)){
+        return Promise.reject({ status: 400, msg: 'bad request' })
+      }
+    const query = `INSERT INTO comments(body, article_id, author)
+    VALUES ($1, $2, $3)
+    RETURNING*`
+    const maxArticleQuery = `SELECT MAX(article_id) FROM articles;`
```

Comment:
I'm not sure this part of your SQL query is needed or really doing anything right now?
If you make a request with an articleID which is a number and doesn't exist you should recieve an empty array.